components/raftstore/src/store/config.rs

```diff
@@ -103,12 +103,17 @@ pub struct Config {
     /// Interval (ms) to check whether start compaction for a region.
     pub region_compact_check_interval: ReadableDuration,
     /// Number of regions for each time checking.
-    pub region_compact_check_step: u64,
+    pub region_compact_check_step: Option<u64>,
```

Comment:
Does it have negative impact on normal workload under these default values? If it does, either make it off by default, or use a more conservative default value so that it won't impact performance

Reply:
According to the oltp_insert result, no performance regression is observed. The default value of testing of region_compact_redundant_rows_percent and  region_compact_check_step are both 10. I modified it to 20 and 5 respectively to reduce their influence on normal workload.